pennylane/measurements/shots.py

```diff
@@ -167,7 +169,7 @@ def __init__(self, shots=None):
         elif isinstance(shots, Sequence):
             if not all(valid_int(s) or valid_tuple(s) for s in shots):
                 raise self._SHOT_ERROR
-            self.__all_tuple_init__([s if isinstance(s, tuple) else (s, 1) for s in shots])
+            self.__all_tuple_init__([s if isinstance(s, Sequence) else (s, 1) for s in shots])
```

Comment:
This change was necessary because the shot vector was getting converted to a list on serialization